app/views/records/show_fields/_abstract.html.erb

```diff
@@ -0,0 +1,5 @@
+<span itemprop="abstract">
+  <%= iconify_auto_link(record.abstract) %>
```

Comment:
Hi Weiwei, iconify does include a sanitize and description does already
have a view.

This is the method which iconify calls:

 /usr/lib64/ruby/gems/2.1.0/gems/rails_autolink-1.1.6/lib/rails_autolink/helpers.rb

This sis the method in sufia for description:

https://github.com/projecthydra/sufia/blob/6.x-stable/app/views/records/show_fields/_description.html.erb



On Tue, Jun 7, 2016 at 12:17 PM, Weiwei Shi <notifications@github.com>
wrote:

> In app/views/records/show_fields/_abstract.html.erb
> <https://github.com/ualbertalib/HydraNorth/pull/1178#discussion_r66124632>
> :
>
> > @@ -0,0 +1,5 @@
> > +<span itemprop="abstract">
> > +  <%= iconify_auto_link(record.abstract) %>
>
> And I think we need one for description as well.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ualbertalib/HydraNorth/pull/1178/files/a60f4ddd696df6d3f2ce57afbd3e3bcca33ae0b8#r66124632>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ADyw15U4KrlEyHAtxBBo7jpdjKmv5Dieks5qJbXTgaJpZM4IwJtY>
> .
>



-- 
Christian Riedlberger
Digital Initiatives Technology
Information Technology Services
christian.riedlberger@ualberta.ca

2-10 Cameron Library
University of Alberta
Edmonton, Alberta
Canada T6G 2J8

phone 780-492-5282
fax 780-492-9243